generator/uscore/uscore_generator.rb

```diff
@@ -247,14 +260,21 @@ def create_must_support_test(sequence)
         test = {
           tests_that: "At least one of every must support element is provided in any #{sequence[:resource]} for this patient.",
           index: sequence[:tests].length + 1,
-          link: 'https://build.fhir.org/ig/HL7/US-Core-R4/general-guidance.html/#must-support',
-          test_code: ''
+          link: 'http://www.hl7.org/fhir/us/core/general-guidance.html#must-support',
+          test_code: '',
+          description: %(
+            US Core Responders SHALL be capable of populating all data elements as part of the query results as specified by the US Core Server Capability Statement.
+            This will look through all #{sequence[:resource]} resources returned from prior searches too see if any of them provide the following must support elements:
```

Comment:
```suggestion
            This will look through all #{sequence[:resource]} resources returned from prior searches to see if any of them provide the following must support elements:
```

Reply:
fixed